From e66bf7bfbb8c7494cdc1b90ebab4b7066a2cb354 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Tue, 4 Feb 2025 17:40:57 +0100
Subject: [PATCH 1638/1695] plat-stm32mp1: conf: default enable
 CFG_STM32_IRQ_NOTIF on stm32mp13x

Enable the driver stm32_irq_notif on stm32mp13x.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 82cc1cb0cfdea4423d470ebd0465c95a83c6972a plat-stm32mp1: conf: default enable CFG_STM32_IRQ_NOTIF on stm32mp13x]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: Id0e2f1e6c0aa2e3f251559fbd72e42549b075f5c
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/440869
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index c1190aa8e..22c7a6357 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -196,6 +196,7 @@ $(call force,CFG_SECONDARY_INIT_CNTFRQ,n)
 $(call force,CFG_STM32_EXTI,y)
 $(call force,CFG_STM32_GPIO,y)
 $(call force,CFG_STM32_HSE_MONITORING,y)
+$(call force,CFG_STM32_IRQ_NOTIF,y)
 $(call force,CFG_STM32_VREFBUF,y)
 $(call force,CFG_STM32MP_CLK_CORE,y)
 $(call force,CFG_STM32MP1_OPTEE_IN_SYSRAM,n)
-- 
2.47.2

